doc/content/modules/user-manual/pages/release-notes/2025.6.0.adoc

```diff
@@ -28,6 +28,9 @@ Publishing a project with a dependency to a library now correctly produces a lib
 image::BindingConnectorAsUsageOnItemUsage.png[BindingConnectorAsUsage between two ItemUsages]
 
 - Fix an issue where some compartments (containing the new element created) were not revealed when the _New Perform Action_ tool was executed.
+- Fix an issue when applying _Add existing nested elements_ on a node containing `TransitionUsages`.
```

Comment:
Please use the past tense when talking about fixes.

Fix an issue where the execution of _Add existing elements _ tool adding `Succession` with guard involving 'start' or 'done' were not well displayed.

and 

Fix an issue where `TransitionUsage` were displayed twice if the _source_ or _target_ of the `TransitionUsage` was displayed multiple times in the same diagram.